cpp/src/plasma/store.cc

```diff
@@ -1207,65 +1211,77 @@ void StartServer(char* socket_name, std::string plasma_directory, bool hugepages
   g_runner->Start(socket_name, plasma_directory, hugepages_enabled, external_store);
 }
 
+void UsageError(const char* error_msg, int exit_code=1) {
+  std::cerr << gflags::ProgramInvocationShortName() << ": " << error_msg << std::endl;
```

Comment:
might be worth a comment on why this doesn't use ARROW_LOG(FATAL)

Reply:
Good point. Added a comment.